old-backup/concept-data-modeling.md

```diff
@@ -0,0 +1,104 @@
+<!-- DO NOT REVIEW -- THIS CONTENT FOR NEXT STAGE
```

Comment:
Like it says on the tin, this content is for the next PR.
It was written during work on this PR whilst conceptualising information.